src/toil/lib/spark.py

```diff
@@ -0,0 +1,414 @@
+# Copyright (C) 2016 UCSC Computational Genomics Lab
+# Copyright (C) 2016 UC Berkeley AMPLab
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Spawns a Spark cluster backed by HDFS.
+
+@author Audrey Musselman-Brown, almussel@ucsc.edu
```

Comment:
We should agree on a policy on these. My position has been to not have any name tags in source files and let the commit history speak about the contributions. We should also have a CONTRIBUTORS file at the project root where we list large contributions.